code/datums/memory/_memory.dm

```diff
@@ -271,7 +271,7 @@
 		/mob/living/basic/statue,
 		/mob/living/basic/stickman,
 		/mob/living/basic/stickman/dog,
-		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher,
+		/mob/living/basic/mining/watcher,
```

Comment:
alphabetize

Reply:
this already is alphabetised, w is after every other preceding letter
unless you are telling me to go "stickman, simple animal, watcher"

Reply:
The alphabet does not have M (as in mining) after S. The subtype takes precedence. One of these lists even has goliath above. The key thing that I want to see is that if someone were to use a vsc auto alphabetize function, it would retain the order. Using the extension would cause an order change

Leverage the mining subtype, you have to unfortunately deal with the after effects which is the fact that it alphabetizes different.